src/acquisition/covid_hosp/common/database.py

```diff
@@ -157,6 +157,10 @@ def insert_dataset(self, publication_date, dataframe):
     sql = f'INSERT INTO `{self.table_name}` (`id`, `{self.publication_col_name}`, {columns}) ' \
           f'VALUES ({value_placeholders})'
     id_and_publication_date = (0, publication_date)
+    NEWLINE="\n"
+    for name, _, _ in self.columns_and_types:
+      if name not in dataframe.columns:
+        raise Exception(f"{name} not in dataframe columns:\n{NEWLINE.join(sorted(dataframe.columns))}")
```

Comment:
how will this behave if we run `fill_covid_hosp_skipped_issue.py` with a file from before all the columns were added? have the hhs files all been back-updated with all the columns?

Reply:
Looking at one of the [old archived files ](https://healthdata.gov/dataset/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/qqte-vkut)from March, it doens't have the new columns, so presumably it'd just error?

Reply:
that should do it?